cli/tests/pcluster/validators/test_ebs.py

```diff
@@ -201,3 +202,27 @@ def test_ebs_volume_size_snapshot_validator(
 
     actual_failures = EbsVolumeSizeSnapshotValidator()(Param(snapshot_id), Param(volume_size))
     assert_failure_messages(actual_failures, expected_message)
+
+
+@pytest.mark.parametrize(
+    "kms_key_id, encrypted, expected_message",
+    [
+        (
+            "arn:aws:kms:us-east-1:012345678910:1234abcd-12ab-34cd-56ef-1234567890ab",
+            None,
+            "Kms Key Id arn:aws:kms:us-east-1:012345678910:1234abcd-12ab-34cd-56ef-1234567890ab "
+            "is specified, the encrypted state can't be None.",
```

Comment:
minor: I'd improve the message saying: "the encrypted state must be True."

Reply:
Done